students/jongho/users/models.py

```diff
@@ -1,19 +1,19 @@
 from django.db import models
 
 class User(models.Model):
-    id         =models.AutoField(primary_key=True)
-    name       =models.CharField(max_length=100)
-    email      =models.EmailField(unique=True)
-    password   =models.CharField(max_length=200)
-    phon       =models.IntegerField(max_length=100)
-    information=models.TextField()
-    created_at =models.DateTimeField(auto_now_add=True)
-    updated_at =models.DateTimeField(auto_now=True)
+    id          = models.AutoField(primary_key=True)
+    name        = models.CharField(max_length=100)
+    email       = models.EmailField(unique=True)
+    password    = models.CharField(max_length=200)
+    phone       = models.CharField(max_length=200)
+    information = models.TextField(blank=True)
+    created_at  = models.DateTimeField(auto_now_add=True)
+    updated_at  = models.DateTimeField(auto_now=True)
 
     class Meta():
       db_table="users"
```

Comment:
코드는 항상 가독성을 생각하고 작성해주세요!
```suggestion
      db_table = "users"
```